multires-lyrics-search/app.py

```diff
@@ -30,7 +29,7 @@ def input_fn():
         reader = csv.reader(f)
         for row in itertools.islice(reader, int(os.environ['JINA_MAX_DOCS'])):
             if row[-1] == 'ENGLISH':
-                d = jina_pb2.Document()
+                d = Document()
```

Comment:
I think you need to do `update_hash` or use `Document` as context manager for it to be called automatically on `__exit__`

Reply:
yes, but will it have any impact in this example?

Reply:
Done!